pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_microcell.py

```diff
@@ -70,14 +79,36 @@ def test_setup(self, n=5):
 
     def test_add_household(self):
         self.microcell.add_people(1)
-        self.microcell.add_household(self.microcell.persons)
+        self.microcell.add_household(self.microcell.persons,
+                                     override_person_id=True)
         self.assertEqual(len(self.microcell.households), 1)
-        household = self.microcell.households[0]
-        self.assertEqual(len(household.persons), 1)
+        original_household = self.microcell.households[0]
+        self.assertEqual(len(original_household.persons), 1)
+        for i in range(len(original_household.persons)):
+            person = self.microcell.persons[i]
+            self.assertEqual(person.id, original_household.id + "." + str(i))
+
+        # Now add another original_household without changing
+        # the id of the person
+        self.microcell.add_household(self.microcell.persons,
+                                     override_person_id=False)
+        self.assertEqual(original_household.persons[0].id,
+                         original_household.id + ".0")
 
     @mock.patch('logging.info')
     def test_logging(self, mock_log):
-        self.microcell.add_household(self.microcell.persons)
+        # Check that the logger is called if there are no people in the
+        # household
+        self.microcell.add_household(self.microcell.persons,
+                                     override_person_id=True)
+        mock_log.assert_called_once()
```

Comment:
Given both tests check that the logging function is called, can we check that it is being called with the right message in each scenario? Should just be a string that matches the log message (with curly brackets in f-strings filled in by the relevant info)

Reply:
Will do